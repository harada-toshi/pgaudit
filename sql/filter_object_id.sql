\! cp sql/filter_object_id/filter_object_id.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/filter_object_id/filter_object_id.sql
\! rm /tmp/config.conf
\! grep "AUDIT:" ./tmp_check/tmp_check.log > results/filter_object_id.out

