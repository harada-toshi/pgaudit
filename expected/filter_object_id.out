	        format = 'AUDIT:(1) %class,%command_tag,%object_type,%object_id,%command_text'
	        format = 'AUDIT:(2) %class,%command_tag,%object_type,%object_id,%command_text'
	        format = 'AUDIT:(3) %class,%command_tag,%object_type,%object_id,%command_text'
LOG:  AUDIT:(1) WRITE,INSERT,TABLE,public.foo,INSERT INTO foo VALUES (1, 'aaa');
LOG:  AUDIT:(2) WRITE,INSERT,TABLE,public.foo,INSERT INTO foo VALUES (1, 'aaa');
LOG:  AUDIT:(3) WRITE,INSERT,TABLE,public.foo,INSERT INTO foo VALUES (1, 'aaa');
LOG:  AUDIT:(1) WRITE,UPDATE,TABLE,public.foo,UPDATE foo SET data = 'AAA' WHERE id = 1;
LOG:  AUDIT:(2) WRITE,UPDATE,TABLE,public.foo,UPDATE foo SET data = 'AAA' WHERE id = 1;
LOG:  AUDIT:(3) WRITE,UPDATE,TABLE,public.foo,UPDATE foo SET data = 'AAA' WHERE id = 1;
LOG:  AUDIT:(1) READ,SELECT,TABLE,public.foo,SELECT * FROM foo;
LOG:  AUDIT:(2) READ,SELECT,TABLE,public.foo,SELECT * FROM foo;
LOG:  AUDIT:(3) READ,SELECT,TABLE,public.foo,SELECT * FROM foo;
LOG:  AUDIT:(1) WRITE,DELETE,TABLE,public.foo,DELETE FROM foo;
LOG:  AUDIT:(2) WRITE,DELETE,TABLE,public.foo,DELETE FROM foo;
LOG:  AUDIT:(3) WRITE,DELETE,TABLE,public.foo,DELETE FROM foo;
